pkg/integrations/v2/register.go

```diff
@@ -419,3 +430,14 @@ func replaceYAMLTypeError(err error, oldTyp, newTyp reflect.Type) error {
 	}
 	return err
 }
+
+// UnmarshalYamlToExporters attempts to convert the contents of yaml string into a set of exporters and then return
+// those configurations.
+func UnmarshalYamlToExporters(contents string) ([]Config, error) {
+	o := &SubsystemOptions{}
+	err := yaml.Unmarshal([]byte(contents), o)
+	if err != nil {
+		return nil, err
+	}
+	return o.Configs, nil
+}
```

Comment:
nit: I probably think this function can be defined unexported in the dynamic loader to keep things tidy.

Reply:
👍